src/clj_uuid.clj

```diff
@@ -457,10 +504,20 @@
   []
   +null+)
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; The MAX UUID Constructor                                    [RFC9562:5.10] ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn max
+  "Generates the maximum UUID, ffffffff-ffff-ffff-ffff-ffffffffffff."
+  ^java.util.UUID
```

Comment:
i dunno.  we're sort of painted into a corner with the nomenclature.  in common lisp maybe it would be analogous to `t` -- true, the top type, opposite of `nil`?  one idea anyway.  maybe a bad one

Reply:
ya fair point